frontend/src/components/PizzaList.js

```diff
@@ -1,22 +1,70 @@
 import Pizza from "./Pizza";
 import { Container, Row, Col } from "react-bootstrap";
 import "bootstrap/dist/css/bootstrap.min.css";
+import { useState, useEffect } from "react";
 
 const PizzaList = (props) => {
   const listOfPizzas = props.listOfPizzas;
```

Comment:
I have not seen this list, want to send a link to the place this is?

Reply:
I sent you an invite to join the project. Please accept.

Reply:
@upulieh I have not seen it? Neither by email nor notifications